src/transport/raw/tests/NetworkTestHelpers.h

```diff
@@ -64,7 +64,7 @@ class LoopbackTransportDelegate
 public:
     virtual ~LoopbackTransportDelegate() {}
 
-    // Called by the loopback transport when it drops a message due to a nonzero mNumMessagesToDrop.
+    // Called by the loopback transport when it drops a message due to a nonzero mNumMessagesToDrop/mNumMessagesToDropSinceIndex
```

Comment:
Please rebase on top of https://github.com/project-chip/connectedhomeip/pull/21321

Reply:
rebased, thanks